src/test/librbd/fsx.cc

```diff
@@ -2437,11 +2437,12 @@ void clone_filename(char *buf, size_t len, int clones)
 
 void clone_imagename(char *buf, size_t len, int clones)
 {
-	if (clones > 0)
-		snprintf(buf, len, "%s-clone%d", iname, clones);
-	else
-		strncpy(buf, iname, len);
+    if (clones > 0) {
```

Comment:
While I don't necessarily like the spacing of `fsx`, can we try to keep it consistent?

Reply:
Does it look ok now? (with 2 spaces as 'coding style' notes). But I do see that it's not consistent with the rest of `fsx` as it has a various indent styles. If you prefer a different indentation, please suggest so. Thanks.

Reply:
Sorry, I was suggesting to just keep the tab indentation in that file for now just to avoid adding more mixed up styles. Alternative, use one commit to convert all the tabs to 2-spaces and then add your cleanup commit on top.

Reply:
No problem. I chose the easier option :)

Reply:
lgtm. one last nit pick, can you squash your two commits down into a single commit? Thx.

Reply:
Done. Thanks!